src/Symfony/Component/Security/Guard/Firewall/GuardAuthenticationListener.php

```diff
@@ -86,7 +88,7 @@ public function handle(GetResponseEvent $event)
         }
     }
 
-    private function executeGuardAuthenticator($uniqueGuardKey, GuardAuthenticatorInterface $guardAuthenticator, GetResponseEvent $event)
+    private function executeGuardAuthenticator($uniqueGuardKey, GuardAuthenticatorLegacyInterface $guardAuthenticator, GetResponseEvent $event)
```

Comment:
@trousers can you please stop it? You're spamming my mailbox. If you want to propose this tool, please open an issue instead.